apps/storybook/stories/public/orama-search-box.stories.tsx

```diff
@@ -25,14 +25,23 @@ const meta: Meta<Components.OramaSearchBox & { preset: keyof DemoIndexConfig }>
         },
       },
     },
+    layout: {
+      options: ['embedded', 'modal'],
```

Comment:
Not sure about `embedded`, can be misleading. Maybe we can rename it as 'in-flow' or 'block' since it corresponds to the default in-flow layout behaviour?

Reply:
I renamed it `embed` (to avoid confusion with `embeddings`), but we can rename it with a better naming later on.